modules/restful_token_auth/plugins/restful/restful_token_auth/token_auth/access_token/1.0/RestfulAccessTokenAuthentication.class.php

```diff
@@ -59,7 +59,21 @@ public function getOrCreateToken() {
 
     $output = $this->viewEntity($id);
 
+    // various options to provide info about the token owner
+    $owner_resource = variable_get('restful_token_auth_owner_resource', '');
+
+    if ($owner_resource !== '<none>') {
+      if ($owner_resource == '<id>') {
+        $output['owner'] = $access_token->uid;
```

Comment:
or even better, we can declare the owner in the ``\RestfulTokenAuthenticationBase::publicFieldsInfo``, probably something like this:

```php
    if (variable_get('restful_enable_users_resource', TRUE)) {
      $public_fields['owner'] = array(
        'property' => 'uid',
        'resource' => array(
          'user' => array(
            'name' => 'users',
            'full_view' => FALSE,
          ),
        ),
      );
    }
    else {
      $public_fields['owner'] = array(
        'property' => 'uid',
      );
    }
```